program-libs/merkle-tree-metadata/README.md

```
@@ -0,0 +1,17 @@
+<!-- cargo-rdme start -->
+
+## light-merkle-tree-metadata
+
+Metadata structs for concurrent, indexed, and batched Merkle trees.
+
+| Module | Description |
+|--------|-------------|
```

markdownlint-cli2 (0.20.0)

Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)